packages/arb-token-bridge-ui/src/components/TransferPanel/OneNovaTransferDialog.tsx

```diff
@@ -78,10 +67,8 @@ export function OneNovaTransferDialog(
             </div>
 
             <BridgesTable bridgeList={fastBridgeList} />
```

Comment:
<img width="668" alt="image" src="https://github.com/OffchainLabs/arbitrum-token-bridge/assets/13184582/759941be-2b22-444d-b34b-db8739924b92">
do we need to change the hover colour here?